query/graphql/src/test/groovy/org/codice/ddf/graphql/GraphQLTransformationTest.groovy

```diff
@@ -24,12 +25,6 @@ class GraphQLTransformationTest extends Specification {
 
```

Comment:
can you use the status codes from here? https://docs.oracle.com/javase/8/docs/api/java/net/HttpURLConnection.html